src/locales/en/common.json

```diff
@@ -2747,6 +2747,11 @@
       "flash-bootloader": "Bootloader updating",
       "flash": "Flashing your device",
       "firmware": "Firmware updating"
+    },
+    "newVersion": "Update Firmware to {{version}} is available",
+    "drawerUpdate": {
+      "title": "Firmaware Update",
```

Comment:
Just noticed it says Firmaware too :D Could you fix that? @klambert-ledger

Reply:
Done 👍